xarray/tests/test_accessor_dt.py

```diff
@@ -234,6 +234,7 @@ def test_dask_accessor_method(self, method, parameters) -> None:
 
     def test_seasons(self) -> None:
         dates = pd.date_range(start="2000/01/01", freq="M", periods=12)
+        dates = dates.append(pd.Index([np.nan, np.datetime64("NaT")]))
```

Comment:
I think testing with `np.datetime64("NaT")` is sufficient:
```suggestion
        dates = dates.append(pd.Index([np.datetime64("NaT")]))
```